_posts/2021-05-02-elasticsearch-basics.md

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Introduction to Elasticsearch Architecture'
+date: 2021-05-02 07:30:00
+---
+
+Elasticsearch is a distributed storage engine optimized for search operations.
+
+Data is stored in Elasticsearch as JSON documents organized in indices. In older versions (prior 7.x) indices was additionaly splitted into types, but they were dropped.
```

Comment:
indices were

"but they were dropped." indices were dropped or the idea of additional split (types)?